lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -215,45 +202,3 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   // Return fully built northstar user object.
   return $ns_user;
 }
-
-
-class Northstar {
```

Comment:
otay. I thought about that when building out aurora and this functionality, but then I felt like I was doing the most